documentation/messages/user.registration.transactional.md

```diff
@@ -0,0 +1,121 @@
+# User registration transactional message
+
+User registration event.
+
+**Routing key**
+
+`user.registration.transactional`
+
+**Exchanges**
+- `transactionalExchange`
+
+**Queues**
+- `activityStatsQueue`
+- `loggingQueue`
+- `mobileCommonsQueue`
+- `transactionalQueue`
+- `userAPIRegistrationQueue`
+- `userRegistrationQueue`
+
+**Parameters**
+
+```js
+{
+
+  /* Required. Always must be `user_register`. */
+  activity: String,
+
+  /* Required. */
+  email: String,
+
+  /* Required. */
+  uid: Number,
+
+  /* Required, Example: `US`. */
+  user_country: String,
+
+  /* Required, Example: `en`. */
+  user_language: String,
+
+  /* Required. Example: `mb-user-register-US`. */
+  email_template: String,
+
+  /* Required. Example: `f2fab1dfd4`. */
+  mailchimp_list_id: String,
+
+  /* Required. Unix timestamp. Example: `881884800`. */
+  birthdate: Number,
```

Comment:
In terms of
> But if we're using Northstar drupal id, should we just get other user fields from it

I'm not sure but is the Northstar `user_id` as the `drupal_id`. I don't think so.

If the optional user values are not provided then yes, I think they should be looked up by a call to Northstar.